pandas/tests/indexes/test_numeric.py

```diff
@@ -451,6 +451,20 @@ def test_astype(self):
             i = Float64Index([0, 1.1, np.NAN])
             pytest.raises(ValueError, lambda: i.astype(dtype))
 
+    @pytest.mark.parametrize("int_dtype", ["uint8", "uint16", "uint32",
+                                           "uint64", "int32", "int64",
+                                           "int16", "int8"])
+    @pytest.mark.parametrize("float_dtype", ["float16", "float32", "float64"])
```

Comment:
Perhaps these (i.e. `int_dtype`, `float_dtype`, `uint_dtype` below) should be `conftest.py` fixtures?

Reply:
yes we have multiple PRs trying to do this (and lots of existing code). can you create / find (I think there is one) an issue about this. PRs to fix even better :> (and may take multiple as using these fixtures can be non-trivial to refactor)

Reply:
I'll add them as fixtures to this PR and then can see afterwards what can be done with them.

Reply:
@jreback : BTW, I didn't realize that the comment in your "review" was referencing the one in this `diff` (I thought I was deleting a duplicate comment).  Oops.